pkg/controller/common/license/crypto.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package license
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+)
+
+const customPadding = 20
+
+var (
+	// aesKey is effectively constant
+	// see https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/java/org/elasticsearch/license/CryptUtils.java#L162-L179
```

Comment:
Should we use [permalinks](https://github.com/elastic/elasticsearch/blob/69cebcd516c208c8e563c0d4e6e9f99e71bf4a3d/x-pack/plugin/core/src/main/java/org/elasticsearch/license/CryptUtils.java#L162-L179) when referring to files that may change ?